docs/home/howto/DATABASES.md

```diff
@@ -7,6 +7,14 @@ Collections, but are supported by PostgreSQL installed from official PostgreSQL
 
 ### PostgreSQL servers
 
+---
+**NOTE**
+
+PgBouncer and replication (Repmgr) extensions are not supported on ARM and these components will not be installed
+even if enabled in configuration. Check ARM design [document](../../design-docs/arm/arm.md)
```

Comment:
I would suggest to not link directly to this document. We need to specify this in more "offical" document later.